Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -4994,8 +5118,11 @@ internal static async Task ListObjectVersions_Test1(MinioClient minio)
                     count += 1;
                 },
                 ex => throw ex,
-                () => { Assert.AreEqual(count, numObjects); });
+                () => { ; });
         }
+
+        Thread.Sleep(1000);
```

Comment:
Generally these are a bad sign, and are inherently flaky.

Reply:
Yes. I am currently working on removing them and I think these thread sleeps cause the notification tests to miss the  notification events intermittently.
But this work is not related to this PR, so I'll work on it as a separate task.